src/logic/map_objects/tribes/tribe_basic_info.cc

```diff
@@ -66,15 +74,22 @@ TribeBasicInfo::TribeBasicInfo(std::unique_ptr<LuaTable> table)
 				std::unique_ptr<LuaTable> script_table = lua.run_script(script_path);
 				script_table->do_not_warn_about_unaccessed_keys();
 				std::set<std::string> tags;
+				std::set<std::string> incompatible_wc;
 				if (script_table->has_key("map_tags")) {
 					std::unique_ptr<LuaTable> t = script_table->get_table("map_tags");
 					for (int key : t->keys<int>()) {
 						tags.insert(t->get_string(key));
 					}
 				}
-				initializations.push_back(Initialization(script_path,
-				                                         script_table->get_string("descname"),
-				                                         script_table->get_string("tooltip"), tags));
+				if (script_table->has_key("incompatible_wc")) {
```

Comment:
That's the same code as above, and as in `gameclient`. Please add a TODO that these three Initialization creation pathways should be pulled out into a common function.